### PR TITLE
✨ feat: add Bulgarian translation

### PR DESCRIPTION
**Pull request by pazoff**
- [x] ✨ feat

#### 🔀 变更说明 | Description of Change

added Bulgarian translation

#### 📝 补充信息 | Additional Information

Bulgarian translation

### CODE REVIEW

**Comment by arvinxx** [comment]
@pazoff hi pazoff ! I'm really glad for your contribution, cloud you pelease check the document [Add New Locale](https://github.com/lobehub/lobe-chat/wiki/Add-New-Locale) ? I think there is some more work need to be done~

**Comment by pazoff** [comment]
> @pazoff hi pazoff ! I'm really glad for your contribution, cloud you pelease check the document [Add New Locale](https://github.com/lobehub/lobe-chat/wiki/Add-New-Locale) ? I think there is some more work need to be done~

bg_BG added to .i18nrc.js

**Comment by arvinxx** [comment]
@pazoff Hi pazoff~ sorry for the delay reply. Could you please add the last and also the most important part for users:

add the locale option:

https://github.com/lobehub/lobe-chat/blob/69fcc78aaa0e39501ce5d490746af81ac7c92d15/src/locales/resources.ts#L107-L114

**Comment by pazoff** [comment]
> @pazoff Hi pazoff~ sorry for the delay reply. Could you please add the last and also the most important part for users —— the Bulgarian locale option:
> 
> https://github.com/lobehub/lobe-chat/blob/69fcc78aaa0e39501ce5d490746af81ac7c92d15/src/locales/resources.ts#L107-L114

added bg_BG

File: src/locales/resources.ts [hunk 1]
```diff
@@ -2,6 +2,7 @@ import resources from './default';
 
 export const locales = [
   'ar',
+  'bg_BG',
```
Comment (arvinxx) [inline src/locales/resources.ts]: it should be `bg-BG`

File: src/locales/resources.ts [hunk 2]
```diff
@@ -112,6 +113,10 @@ export const localeOptions: LocaleOptions = [
     label: 'Tiếng Việt',
     value: 'vi-VN',
   },
+  {
+    label: 'Bulgarian',
```
Comment (arvinxx) [inline src/locales/resources.ts]: Why not use the Bulgarian language to display "Bulgarian" ?

File: src/locales/resources.ts [hunk 3]
```diff
@@ -112,6 +113,10 @@ export const localeOptions: LocaleOptions = [
...
+    value: 'bg_BG',
```
Comment (arvinxx) [inline src/locales/resources.ts]: bg-BG

**Review by arvinxx — CHANGES_REQUESTED** [review]
some detail need to be polished

**Review by arvinxx — APPROVED** [review]
LGTM now, I'll merge it. Thanks for your contribution!

**Pull request merged**